scalableminds/webknossos
# Hide advanced "dataset" tab for non-admin users

### Mailable description of changes:
- [Dashboard] Hid `Dataset`tab for non-admin users

### Steps to test:
- Login as an admin. Dashboard.Dataset should be there
- Login as a regular user. Dashboard.Dataset should not be there
- Go to Users-->Show Tracings. `Dataset` tab should not be there in either of the above cases.

### Issues:
- fixes #2038 

------
